linters/README.md

```diff
@@ -5,3 +5,5 @@ Files to configure the linters we use for each language and technology
 
 * Ruby
   * [Rubocop](/linters/ruby/.rubocop.yml)
+* JavaScript
```

Comment:
What about `JavaScript (Babel + React)`? I'm using JS but not babel or react, and even though we have some (maybe most) of our projects using those technologies, this is not a JS linter if it has constraints that don't apply to every JS project.